<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptStatusAuto</class>
 <widget class="QWidget" name="OptStatusAuto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Auto status</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="_2">
        <property name="margin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>6</number>
        </property>
        <item row="2" column="1">
         <widget class="QSpinBox" name="sb_asOffline">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="suffix">
           <string> minutes</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="sb_asXa">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="suffix">
           <string> minutes</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="ck_asOffline">
          <property name="text">
           <string>Auto offline after:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="ck_asXa">
          <property name="text">
           <string>Auto XA after:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="ck_asAway">
          <property name="text">
           <string>Auto away after:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="sb_asAway">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="suffix">
           <string> minutes</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLineEdit" name="le_asPriority">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLineEdit" name="le_asXaPriority">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="lb_asPriority">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Priority:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="lb_asXaPriority">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Priority:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="TextLabel1_8">
        <property name="text">
         <string>Status message:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ChatEdit" name="te_asMessage">
        <property name="acceptRichText">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>32767</width>
          <height>100</height>
         </size>
        </property>
        <property name="tabChangesFocus">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_defPriority">
     <property name="title">
      <string>Default priority</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="lb_dpOnline">
            <property name="text">
             <string>Online:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="sb_dpOnline">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="minimum">
             <number>-128</number>
            </property>
            <property name="maximum">
             <number>127</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="2">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="lb_dpAway">
            <property name="text">
             <string>Away:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="sb_dpAway">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="minimum">
             <number>-128</number>
            </property>
            <property name="maximum">
             <number>127</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="lb_dpChat">
            <property name="text">
             <string>Free For Chat:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="sb_dpChat">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="minimum">
             <number>-128</number>
            </property>
            <property name="maximum">
             <number>127</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="lb_dpXa">
            <property name="text">
             <string>XA:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="sb_dpXa">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="minimum">
             <number>-128</number>
            </property>
            <property name="maximum">
             <number>127</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="lb_dpDnd">
            <property name="text">
             <string>Do Not Disturb:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="sb_dpDnd">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="minimum">
             <number>-128</number>
            </property>
            <property name="maximum">
             <number>127</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="2">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="lb_dpInvisible">
            <property name="text">
             <string>Invisible:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="sb_dpInvisible">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="minimum">
             <number>-128</number>
            </property>
            <property name="maximum">
             <number>127</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ChatEdit</class>
   <extends>QTextEdit</extends>
   <header>msgmle.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>ck_asAway</tabstop>
  <tabstop>sb_asAway</tabstop>
  <tabstop>le_asPriority</tabstop>
  <tabstop>ck_asXa</tabstop>
  <tabstop>sb_asXa</tabstop>
  <tabstop>le_asXaPriority</tabstop>
  <tabstop>ck_asOffline</tabstop>
  <tabstop>sb_asOffline</tabstop>
  <tabstop>te_asMessage</tabstop>
  <tabstop>sb_dpOnline</tabstop>
  <tabstop>sb_dpChat</tabstop>
  <tabstop>sb_dpAway</tabstop>
  <tabstop>sb_dpXa</tabstop>
  <tabstop>sb_dpDnd</tabstop>
  <tabstop>sb_dpInvisible</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>ck_asAway</sender>
   <signal>toggled(bool)</signal>
   <receiver>sb_asAway</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>28</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ck_asXa</sender>
   <signal>toggled(bool)</signal>
   <receiver>sb_asXa</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ck_asOffline</sender>
   <signal>toggled(bool)</signal>
   <receiver>sb_asOffline</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ck_asAway</sender>
   <signal>toggled(bool)</signal>
   <receiver>lb_asPriority</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ck_asAway</sender>
   <signal>toggled(bool)</signal>
   <receiver>le_asPriority</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ck_asXa</sender>
   <signal>toggled(bool)</signal>
   <receiver>lb_asXaPriority</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ck_asXa</sender>
   <signal>toggled(bool)</signal>
   <receiver>le_asXaPriority</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
